cfme/infrastructure/pxe.py

```diff
@@ -574,6 +574,39 @@ def delete(self, cancel=False, *ct_objs):
             else:
                 navigate_to(ct_obj, 'Details')
 
+    def copy(self, ct_obj, name=None, description=None, cancel=False):
+        """
+         This method is used to copy a Customization Template server via UI.
+
+         Args:
+             ct_obj : This field is the object of Customization Template.
+             name (str): This field contains the name of the newly copied Customization Template.
+             description (str) : This field contains the description of the newly
+              copied Customization Template.
+             cancel (bool): It's used for flag to cancel or not the copy operation.
+        """
+
+        view = navigate_to(ct_obj, 'Details')
+        view.toolbar.configuration.item_select("Copy this Customization Template")
```

Comment:
**REQUIRED:** The preference here is to have a new navigation destination I think, that will be linked to the AddView. This way you don't have to go to the trouble of instantiating the view for yourself.